jupyter_client/asynchronous/client.py

```diff
@@ -0,0 +1,375 @@
+"""Implements an async kernel client"""
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from __future__ import print_function
```

Comment:
I don't think this is necessary now that we drop 2.7.

Reply:
Thanks, removing it.